libs/gisday/platform/ngx/core/src/lib/pages/competitions/art/art.component.html

```diff
@@ -44,7 +44,7 @@ <h1 class="trailer-1">Texas A&M University Art Trail Challenge</h1>
 
         <h2 class="trailer-1 leader-2">How to participate</h2>
         <ul class="numbered-custom">
-          <li>Download the official TxGIS Day Competitions App on your mobile device.</li>
+          <li>Download and install the <a href="/app">Official TxGIS Day Competitions App</a> on your mobile device.</li>
```

Comment:
This is a very simple redirect, so native href is perfectly fine. As a matter of doing things the "Angular way" and future reference, use routerLink

https://v17.angular.io/api/router/RouterLink